ui/admin/app/styles/_notify.scss

```diff
@@ -86,7 +86,7 @@
   }
 }
 
-@media only screen and (max-width: 680px) {
+@media only screen and (width <= 680px) {
```

Comment:
what about other places where we use sizing variables? `(max-width: media.width(small)` Do we need to take care of these as well?

Reply:
@DhariniJeeva that's a great point.  Those didn't show up on linting, but the pattern should be the same .  Let me do some testing

Reply:
To try answer @DhariniJeeva question, here my 2 cents: (@dvonanderson please, let me know if I am missing something)

- The change from `max-widdth: X` to `width <= X` it just applies to media-queries. Not sure the motivation behind this change, perhaps linter suggestion. I do personally think, is easier to understand now.
- The change, responds to a new syntax introduced within Media queries Level 4, [here the documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries#syntax_improvements_in_level_4).
- This change, should NOT be applied outside of `@media-queries`.

Reply:
After a chat with @dvonanderson and as I mentioned above, Yes, we have other `@media` queries (the ones mentioned by @DhariniJeeva ) that need to be change. Devon is working on it and will provide those changes within this PR 😉

Reply:
@calcaide @DhariniJeeva just pushed up new commit that has all `@media` queries switched over to use `prefix` notation.  Used a red background to test.

https://github.com/hashicorp/boundary-ui/assets/24277002/0d573314-96d5-43ce-860f-3cd35d38595c